boxsdk/object/trash.py

```diff
@@ -0,0 +1,120 @@
+# coding: utf-8
+from __future__ import unicode_literals, absolute_import
+
+import json
+
+from .base_endpoint import BaseEndpoint
+from ..pagination.limit_offset_based_object_collection import LimitOffsetBasedObjectCollection
+
+
+class Trash(BaseEndpoint):
+    """Box API endpoint for performing trash related actions in Box."""
+
+    def get_from_trash(self, item, fields=None):
```

Comment:
This method should probably have a better name — `client.trash().get_from_trash()` stutters a bit